clients/ts/FunctionalTests/ts/HubConnectionTests.ts

```diff
@@ -606,8 +606,8 @@ describe("hubConnection", () => {
     it("transport falls back from WebSockets to SSE or LongPolling", async (done) => {
```

Comment:
This function is not relevant in node, you can just disable it. And then you can use `window` directly.